applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -588,9 +588,13 @@ static void gnss_event_handler(int event)
 		LOG_INF("GNSS_EVT_FIX");
 		on_gnss_evt_fix();
 		break;
-	case NRF_MODEM_GNSS_EVT_AGPS_REQ:
-		LOG_INF("GNSS_EVT_AGPS_REQ");
-		/* This will send A-GNSS/P-GPS requests only if cloud assistance is enabled. */
+	case NRF_MODEM_GNSS_EVT_NMEA:
+		if (IS_ENABLED(CONFIG_SLM_LOG_LEVEL_DBG)) {
+			on_gnss_evt_nmea();
+		}
+		break;
```

Comment:
This code should be deleted. Case `NRF_MODEM_GNSS_EVT_NMEA` has been moved a few lines up.

Reply:
Yes, updated now.